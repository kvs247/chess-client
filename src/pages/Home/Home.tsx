import { FC } from "react";
import { useGetTestQuery } from "../../store/test/api";
import { TestData } from "../../store/test/types";
import ChessBoard from "../../components/ChessBoard/ChessBoard";

const startingFEN = "rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1";

const Home: FC = () => {
  const { data, isError, isLoading } = useGetTestQuery(null);

  let response: TestData[] | undefined;  
  if (!isError && !isLoading && data !== undefined) {
    response = data.response
  }  

  return (
    <div style={{ backgroundColor: "grey" }}>
      <h1>Welcome to Chess Is Hard by Kyle</h1>
      <ChessBoard 
        fen={startingFEN}
      />
    </div>
  );
};

export default Home;