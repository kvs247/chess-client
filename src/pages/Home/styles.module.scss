.Wrapper {
  background-color: grey;
  height: 100vh;
  overflow: hidden;
}

.Button {
  position: absolute;
  top: 20px;
  left: 20px;
}

.ChessBoardWrapper {
  display: flex;
  justify-content: center;
  padding-top: calc(10vh / 2);
  padding-bottom: calc(10vh / 2);
}

