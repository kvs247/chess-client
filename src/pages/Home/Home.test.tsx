import { render, cleanup } from "@testing-library/react";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import Home from "./Home";

afterEach(cleanup);

it("renders", () => {
  const initialState = {
    testApi: {
      queries: {
        "getTest(null)": {
          status: "fulfilled",
          endpointName: "getTest",
          requestId: "absdgvas7dv8gsa",
          originalArgs: null,
          startedTimestamp: 0,
          data: { response: [] },
          fulfilledTimestamp: 1,
        }, 
      },
    }, 
  };

  const mockStore = configureStore();
  const store = mockStore(initialState);

  const { asFragment } = render(
    <Provider store={store}>
      <Home />
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot();  
});
