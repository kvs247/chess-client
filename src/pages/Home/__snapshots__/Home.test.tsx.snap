// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snaphot with data 1`] = `
<div
  style="background-color:grey"
>
  <h1>
    Welcome to Chess Is Hard by Kyle
  </h1>
  <div
    class="Wrapper"
    id="board-wrapper"
  >
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-queen.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-king.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-queen.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-king.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
  </div>
</div>
`;

exports[`Snaphot without data 1`] = `
<div
  style="background-color:grey"
>
  <h1>
    Welcome to Chess Is Hard by Kyle
  </h1>
  <div
    class="Wrapper"
    id="board-wrapper"
  >
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-queen.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-king.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="black-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src=""
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-pawn.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-queen.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-king.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-bishop.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:green"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-knight.png"
        style="transform:translate(0px,0px)"
      />
    </div>
    <div
      class="Square"
      style="background-color:white"
    >
      <img
        alt=""
        class="PieceImg react-draggable"
        draggable="false"
        src="white-rook.png"
        style="transform:translate(0px,0px)"
      />
    </div>
  </div>
</div>
`;
